Documentation/ceph-object-bucket-claim.md

```diff
@@ -105,13 +105,14 @@ provisioner: rook-ceph.ceph.rook.io/bucket [2]
 parameters: [3]
   objectStoreName: my-store
   objectStoreNamespace: rook-ceph
-  region: us-west-1
-  bucketName: ceph-bucket [4]
-reclaimPolicy: Delete [5]
+  region: us-west-1 [4]
+  bucketName: ceph-bucket [5]
+reclaimPolicy: Delete [6]
 ```
 1. `label`(optional) here associates this `StorageClass` to a specific provisioner.
 1. `provisioner` responsible for handling `OBCs` referencing this `StorageClass`.
 1. **all** `parameter` required.
+1. `region` defines the region in which the bucket should be created. For Rook, this is CephObjectZoneGroup (if applicable) of the related CephObjectStore; So if you need to define region other than `us-east-1` create `CephObjectZoneGroup` accordingly. Please refer to [Ceph's documentation](https://docs.ceph.com/en/latest/radosgw/multisite/#zone-groups) for more details. 
```

Comment:
Is the name of the CephObjectZoneGroup CR the name of the `region`? If so, how do we get `us-east-1` by default since the default zone group name is the same as the object store? Maybe an example yaml would help show how to create a region named `us-west-1` to match the region in the storage class.

Reply:
@travisn `us-east-1` is the default hard corded value inside [rgw code base](https://github.com/ceph/ceph/blob/master/src/rgw/rgw_rest_client.cc#L319), most likely default zone group set by rook may also work(aka the name of object store). But people tend to use the region as `us-east-1` for was clients. Hence I mentioned about `us-east-1` other than the zone group value set by the rook. Or in other words, if the `us-east-1` it may pick up the default zone group value or else they try to find out the corresponding zonegroup

Reply:
I'm still not clear... So if they want to have the `region: us-west-1` as in this example, how would the object store, zone group, etc need to be created? Perhaps we could modify the [object-multisite.yaml](https://github.com/rook/rook/blob/master/deploy/examples/object-multisite.yaml) example to use the `us-west-1` region to match this example?

Reply:
@travisn : What about having proper documentation in ceph docs then only this PR need to be merged, instead of modifying the `object-mutlisite.yaml`. If `object-mutlisite.yaml` is modified then that may create unnecessary confusion for users who only intended to use mutlisite.

Reply:
Sounds good to include a yaml snippet here in the docs that shows how to create the object store with a non-default region.